nixos/modules/services/networking/cloudflared.nix

```diff
@@ -193,6 +219,16 @@ in
                 };
               };
 
+              certificateFile = lib.mkOption {
+                type = with lib.types; nullOr path;
+                description = ''
+                  Cert.pem file.
+
+                  See [Cert.pem](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-useful-terms/#certpem).
+                '';
+                default = null;
+              };
+
```

Comment:
this `certificateFile` block is duplicated

Reply:
fixed